socorro/processor/mozilla_processor_2015.py

```diff
@@ -63,6 +63,7 @@
         "socorro.processor.signature_utilities.SignatureRunWatchDog, "
         "socorro.processor.signature_utilities.SignatureIPCChannelError, "
         "socorro.processor.signature_utilities.SignatureIPCMessageName, "
+        "socorro.processor.signature_utilities.SigTrim, "
```

Comment:
I like you proposal Will, and I think it makes sense. I'll merge this new rule into the ``SigTrunc`` one.